evalml/tests/pipeline_tests/regression_pipeline_tests/test_baseline_regression.py

```diff
@@ -2,7 +2,47 @@
 import pandas as pd
 from pandas.testing import assert_series_equal
 
-from evalml.pipelines import BaselineRegressionPipeline
+from evalml.pipelines import (
+    BaselineRegressionPipeline,
+    MeanBaselineRegressionPipeline
+)
+
+
+def test_baseline_regression_init(X_y_binary):
```

Comment:
Do you think these tests can move up the inheritance hierarchy?  Like maybe to PipelineBase?  I feel like I'm missing some context on special behaviors of some of the different pipeline child classes as to which exhibit what special behavior, but this seems to be universal to pipelines.  Let me know what you think and I can file an issue if you agree.

- [ ] @chukarsten file an issue if this testing behavior needs to move up

Reply:
Added some tests for `BinaryClassificationPipeline`, `MulticlassClassificationPipeline`, `RegressionPipeline` but kept these since they test for our specific baseline classes.